sentry/src/main/java/io/sentry/SentryTracer.java

```diff
@@ -86,6 +86,12 @@ public SentryTracer(
       this.baggage = new Baggage(scopes.getOptions().getLogger());
     }
 
+    final @NotNull SentryId continuousProfilerId =
```

Comment:
just a thought: I'd imagine this would be better leaving on the Scope(s), but since it's a global profiler it's probably fine.